File: Java/Mahasiswa.java
class Mahasiswa extends SivitasAkademik {
    private String NIM;
    private String fakultas;
    private String prodi;

    Mahasiswa() {

    }

    Mahasiswa(String NIM, String fakultas, String prodi) {
        this.NIM = NIM;
        this.fakultas = fakultas;
        this.prodi = prodi;
    }

    public void setNIM(String NIM) {
        this.NIM = NIM;
    }

    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }

    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    public String getNIM() {
        return this.NIM;
    }

    public String getFakultas() {
        return this.fakultas;
    }

    public String getProdi() {
        return this.prodi;
    }
}

File: Python/Mahasiswa.py
from SivitasAkademik import SivitasAkademik


class Mahasiswa(SivitasAkademik):
    __NIM = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, NIM="", prodi="", fakultas=""):
        self.__NIM = NIM
        self.__prodi = prodi
        self.__fakultas = fakultas

    def getNIM(self):
        return self.__NIM

    def getProdi(self):
        return self.__prodi

    def getFakultas(self):
        return self.__fakultas

    def setNIM(self, NIM):
        self.__NIM = NIM

    def setProdi(self, prodi):
        self.__prodi = prodi

    def setFakultas(self, fakultas):
        self.__fakultas = fakultas
